examples/kitchen-sink-cra/src/index.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
delete this file?

Reply:
Could. It really doesn't matter design wise. But it is used.